### PR TITLE
Add information about README/CONTRIBUTING. Closes #3

### LINKED ISSUE #3: Improve docs on how to consume the renderer and how to contribute to it
Improve project docs to make it easier for others to consume and to contribute

- [ ] CONTRIBUTING with lint/test/building
- [ ] README with information about philosophy

### PR DESCRIPTION
**Pull request by brendo**
I've tried to include as much detail as I think we have at the moment without making any promises we can't keep. I think it's a little soon for contributions, but I definitely don't want to discourage any either!

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,5 +1,34 @@
 # Contributing to Open API Renderer
 
+Thanks being interested in contributing to this project! It's Temando's first open source project so bare with us while we iron out the kinks. The following describes some basic information that should help you get started.
```
Comment (ghost) [inline CONTRIBUTING.md]: "Thank you for your interest..." or "Thanks for being interested...."

Comment (ghost) [inline CONTRIBUTING.md]: I'd say "first major open source project".

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,5 +1,34 @@
...
+
+## Browser/device support
+
+This project aims to support evergreen browsers, such as Chrome and Firefox. Support for IE and Safari versions is to be determined, but likely will start IE 11/Safari 9.
```
Comment (ghost) [inline CONTRIBUTING.md]: You might consider Safari evergreen. They release updates independent of the OS: https://support.apple.com/en-au/HT201222 (on the desktop, anyway).

Comment (ghost) [reply CONTRIBUTING.md]: Safari 10 has been out for a while: https://support.apple.com/en-au/HT207157 for the last three Mac OS releases. Probably a fine minimum.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,5 +1,34 @@
...
+
 ## Linting
 
 This project is written adhering to the [StandardJS](https://standardjs.com/) code style. [ESLint](http://eslint.org/) is used to enforce [`standard`](https://github.com/feross/eslint-config-standard) and [`standard-react`](https://github.com/feross/eslint-config-standard-react) rule sets.
+
+## Testing
+
+This project uses [Jest](https://facebook.github.io/jest/) for testing. Tests should be committed with each change to ensure regressions do not occur.
```
Comment (ghost) [inline CONTRIBUTING.md]: Could clarify "code change".

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,5 +1,34 @@
...
+
+> Test coverage is not currently enforced, but it will in future.
```
Comment (ghost) [inline CONTRIBUTING.md]: "... but it will be in the future."

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,5 +1,34 @@
...
+
+## Github & Merge Requests
+
+> We prefer small, focused MR's rather than large complex MR's whenever possible.
+
+We use [Github projects](https://github.com/temando/open-api-renderer/projects) to manage phases of development. If you'd like to chip in:
+
+1. Look at the *TODO* column for unassigned work. Generally it's in priority order.
```
Comment (ghost) [inline CONTRIBUTING.md]: "1. Open a project and look at..."

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1,5 +1,34 @@
...
+1. Leave a note on the issue saying you'd like to tackle it.
+1. Create a new branch in your fork and begin!
+1. When raising the MR, include as much detail as possible to helper the reviewer
```
Comment (ghost) [inline CONTRIBUTING.md]: "... to help ..."

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -1,5 +1,34 @@
...
+
+Alternatively, sometimes we have ideas that we are not sure how to approach, or don't fit in with our priorities. These issues are labelled as [wishes](https://github.com/temando/open-api-renderer/issues?q=is%3Aissue+is%3Aopen+label%3Awish). Feel free to tackle one of these or contribute any feedback.
+
+## Versioning
+
+The project will, in future, abide to [semver](http://semver.org/) and keeping a [CHANGELOG](http://keepachangelog.com/en/0.3.0/). Right now we're at the peak of development and until we reach a release, these processes would add extra overhead that we are not willing to absorb just yet.
```
Comment (ghost) [inline CONTRIBUTING.md]: Just "keep a changelog".

File: README.md [hunk 8]
```diff
@@ -44,3 +44,11 @@ yarn run build:prod
 ```
 
 Deploy the files at `/dist` to your server.
+
+## Philosophy
+
+While this project is currently focused on visualising Open API V3 specifications, it is architectured in such a way that the React components deal with a `UIReadySchema`, which is agnostic (but heavily based on Open API V3) of specification.
```
Comment (ghost) [inline README.md]: Alternative to "agnostic", "generic"?

File: README.md [hunk 9]
```diff
@@ -44,3 +44,11 @@ yarn run build:prod
...
+
+The dream is that this renderer could visualise other formats by introducing new parsers which transform documents into the common `UIReadySchema` format. This approach allows us to build something sustainable and scalable, where the community can help contribute new parsers (among other things!) as required.
+
+The project is definitely in it's infancy and we are not ready for that yet, but we hope by sharing this vision early, others can help us make it a reality. For contributing information, see [CONTRIBUTING.md](CONTRIBUTING.md).
```
Comment (ghost) [inline README.md]: "its"

**Review by ghost — CHANGES_REQUESTED** [review]
Review complete.

File: CONTRIBUTING.md (hunk 1 above)
Comment (quangkhoa) [reply CONTRIBUTING.md]: `bare` => `bear`

File: README.md (hunk 8 above)
Comment (quangkhoa) [reply README.md]: `architectured` => `architected`

**Pull request merged**